services/state/common.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+import "basic_types.proto";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.common">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * A single 64-bit number identifying a Hedera native entity.
+ */
+message EntityNumber {
+    int64 number = 1;
+}
+
+/**
+ * Pair of AccountID and TokenID to represent TokenRelation
```

Comment:
Nit: if we're calling this message `EntityIDPair`, we should change this javadoc to reflect that it can be a pair of any entity IDs. If this is only for token relations, why not call it `TokenRelationID` or similar? 

Not a blocker

Reply:
@jsync-swirlds is updating the documentation. Please send suggestions to him.

Reply:
The items in this file aren't in use.  There is a follow-on change in progress that removes this file entirely (or it could be done here).